subsys/bluetooth/host/gatt.c

```diff
@@ -1727,6 +1729,13 @@ static int gatt_unregister(struct bt_gatt_service *svc)
 		if (is_host_managed_ccc(attr)) {
 			gatt_unregister_ccc(attr->user_data);
 		}
+
+		/* The stack should not clear any handles set by the user. */
+		if (attr->_auto) {
```

Comment:
Maybe we can change the name `_auto` to something more descriptive, so we don't need the comment here?

Reply:
Suggestions?

Reply:
Naming is hard :sweat: Something like the following?

- `managed_handle`
- `dynamic_handle`
- `automatic_handle`
- `stack_issued_handle`
- `auto_assigned_handle`
- `auto_generated_handle`

I like `auto_assigned_handle` best.